lib/helpers.js

```diff
@@ -67,6 +67,26 @@ function getConfig() {
       device: {}
     };
   }
+  try {
+    fs.statSync(__dirname + '/../tmp/store.json');
```

Comment:
this will conflict vs master

`git rebase master` from this branch or fix manually. ask me or @diegoribero for help with rebase.